README.md

```diff
@@ -6,6 +6,10 @@
 
 `swift package generate-xcodeproj`
 
+Open the project and run the Commands target. The target is configured to run an example with Ruby IRB.
```

Comment:
Let's either:

- spell out IRB fully
- add a URL to some more information on it.

Reply:
I don't know what IRB stands for.. something with interactive probably.. and Ruby... I will fix it.

Reply:
> The abbreviation irb comes from the fact that the filename extension for Ruby is ".rb", although interactive Ruby files do not have an extension of ".irb".

— https://en.m.wikipedia.org/wiki/Interactive_Ruby_Shell